electron/main.js

```diff
@@ -416,7 +416,7 @@ class ElectronWrapperInit {
       }
     };
 
-    app.on('web-contents-created', (event, contents) => {
+    app.on('web-contents-created', (createdEvent, contents) => {
```

Comment:
@ffflorian if you have a better name.. let me know! because of the shadowing the line 422

Reply:
I like it